packages/graphql-server/src/resolvers/index.ts

```diff
@@ -11,8 +11,8 @@ import { login } from './auth/resolvers'
 
 export const resolvers = {
   Query: {
-    contact: queryContact,
-    contacts: queryContacts,
+    getContact: queryContact,
```

Comment:
It should use the name on Query when you fetch `swager-to-graphql` because it have to consistent with the API term